specification/k3po.control/SPEC.md

```diff
@@ -29,60 +47,20 @@ If the "content-length" header is specified, then a message body of the specifie
 
 Headers that are not understood / processed by a command must be ignored.
 
-
-### Flow
-
-The communication between the test framework and the driver is an asynchronous conversation - the test framework sends commands, the driver 
-may respond with one or more events or it may not respond at all (no event sent) when the action is finished.
-
```

Comment:
I added an introduction to address this point.  I think what you are getting at here is that it would be a mistake to think of the protocol as request and response.  So I kept the async part and explained events are sent by state changes